pallets/creditcoin/src/migrations/v8.rs

```diff
@@ -0,0 +1,100 @@
+use super::Vec;
+use super::WeightInfo;
+use super::{AccountIdOf, Config};
+use super::{Migrate, StorageVersion, Weight};
+use frame_support::{
+	storage::migration::move_storage_from_pallet,
+	storage_alias,
+	traits::{GetStorageVersion, PalletInfoAccess, StorageInstance},
+	Blake2_128Concat, StoragePrefixedMap,
+};
+use sp_io::hashing::twox_128;
+use sp_runtime::traits::SaturatedConversion;
+use sp_std::{marker::PhantomData, vec};
+
+pub static SCHEDULER_PREFIX: &str = "TaskScheduler";
```

Comment:
It's worth noting that the correctness of this migration relies on the fact that the task scheduler pallet is given this name in `construct_runtime!`. If we were to change the name there, the migration would move the authorities to no mans land (i.e. a storage address for a nonexistent pallet).

Reply:
That's why the PR comes with two unit tests. If these invariants are not satisfied lmk to rework the tests.